test/context.py

```diff
@@ -0,0 +1,3 @@
+import sys
```

Comment:
I don't quite understand what the empty inits and the context are doing, so maybe we could go over that again on Sunday? It sounds like it may have been something that could make our testing better/easier/more typical?

Reply:
Yeah, we can talk on Sunday!

I actually didn't mean to include the context file in the pull request-- it isn't doing anything. But I was experimenting with [this structure](http://docs.python-guide.org/en/latest/writing/structure/) from a Kenneth Reitz post.

The empty `__init__.py` file initializes a source code directory into a package, which can make importing easier, but I don't think it is really doing anything at the moment... [Here's a post to peak at.](http://mikegrouchy.com/blog/2012/05/be-pythonic-__init__py.html)

Thanks for taking a close look at the changes!!